packages/core/src/app/ovm/state-manager.ts

```diff
@@ -24,13 +24,10 @@ import { getOverlappingRange, ONE, rangesIntersect } from '../../app'
  * See: http://spec.plasma.group/en/latest/src/05-client-architecture/state-manager.html for more details.
  */
 export class DefaultStateManager implements StateManager {
-  private stateDB: StateDB
-  private pluginManager: PluginManager
-
-  public constructor(stateDB: StateDB, pluginManager: PluginManager) {
-    this.stateDB = stateDB
-    this.pluginManager = pluginManager
-  }
+  public constructor(
+    private stateDB: StateDB,
+    private pluginManager: PluginManager
+  ) {}
```

Comment:
Does this automatically set these params to private variables within this class?

Reply:
Yep 😄

Reply:
oooh I didn't know this 😄!